.github/workflows/mediasoup-node.yaml

```diff
@@ -11,12 +11,22 @@ concurrency:
 jobs:
   ci:
     strategy:
+      fail-fast: false
       matrix:
         ci:
           - os: ubuntu-20.04
             node: 18
           - os: ubuntu-22.04
             node: 20
+          - os: ubuntu-24.04
+            node: 20
+            meson_args: '-Db_sanitize=address'
+          - os: ubuntu-24.04
+            node: 20
+            meson_args: '-Db_sanitize=undefined'
+          - os: ubuntu-24.04
+            node: 20
+            meson_args: '-Db_sanitize=thread'
```

Comment:
What is the rationale for having 3 separate matrix entries with different `-Db_sanitize` options? Why not all together into a single one?

Reply:
Only one sanitizer can be enabled at a time.
For example, thread sanitizer and address sanitizer cannot be enabled at the same time.

Reply:
Then why this?

```
clang++ -fsanitize=signed-integer-overflow,null,alignment -fno-sanitize-recover=null -fsanitize-trap=alignment a.cc
```

https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html

Reply:
I'm testing `-fsanitize='undefined,address,thread` and it definitely works. So honestly I don't think that we need to run just one at the same time.

Reply:
ok, but the I read this:

https://stackoverflow.com/a/68907718